packages/select/src/FilterMultiSelect/provider/MenuTriggerProvider/MenuTriggerProvider.spec.tsx

```diff
@@ -10,9 +10,15 @@ import {
 } from "./MenuTriggerProvider"
 
 const MenuTriggerProviderWrapper = ({
+  isOpen,
   defaultOpen,
+  onOpenChange,
 }: Partial<MenuTriggerProviderProps>) => (
-  <MenuTriggerProvider defaultOpen={defaultOpen}>
+  <MenuTriggerProvider
+    defaultOpen={defaultOpen}
+    isOpen={isOpen}
+    onOpenChange={onOpenChange}
```

Comment:
If you want to save some lines

```suggestion
(props: Partial<MenuTriggerProviderProps>) => (
  <MenuTriggerProvider {...props} />
```